app/static/Climate-Mind_bundled.yml

```diff
@@ -235,7 +250,7 @@ paths:
                     userB:
                       name: Bob
                     conversationStatus: 2
-                    consent: true
+                    consent: false
```

Comment:
Is this one meant to say consent= true?

Reply:
No, this is under the "Successful GET Conversation Response Body with consent=false" example.